src/shared/firewall-util-nft.c

```diff
@@ -658,6 +674,10 @@ static int fw_nftables_init_family(sd_netlink *nfnl, int family) {
         if (r < 0)
                 goto out_unref;
 
+        if (family == AF_INET6) {
```

Comment:
maybe add an assert(IN_SET(family, AF_INET, AF_INET6)) or so to the top of function, which acts as kind of encoded documentation what this supports and that if family is not AF_INET6 then it is AF_INET